### ISSUE TITLE
[DOCUMENT] GIBCT SCO - Lists are not picked up correctly by screen readers in Section 305 PDF

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3869]
## Description

VA 508 office has identified the [Colmery Act Section 305](https://www.benefits.va.gov/gibill/docs/scotraininginstructions.pdf) PDF as having lists that do not announce properly to screen readers. Screenshot attached below is illustrative of one list of many that need revised in this PDF.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Remediation Resources
* Run the Common Look plugin to identify issues
* Re-tag the document in Acrobat X and ensure the lists are tagged as `Lbl` with `LI` items insie them. ( See https://www.adobe.com/accessibility/products/acrobat/pdf-repair-add-tags.html ) for more information.

## Acceptance Criteria

* Lists are announced properly, announcing how many items are included
* List items are announced properly

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

<img width="511" alt="Screen Shot 2019-12-02 at 5 10 00 PM" src="https://user-images.githubusercontent.com/934879/70003092-bc94d600-1527-11ea-8492-e972ee569a07.png">

**Comment by desireeturner** [comment]
@1Copenut The Education team has been informed of this issue.

**Comment by TamGIBill** [comment]
Corrected 508 issues.

**Comment by 1Copenut** [comment]
Thank you Desiree. I sent the update to the VA 508 office.

**Comment by 1Copenut** [comment]
@TamGIBill I heard back from the VA 508 office regarding this document. The list issue looks to be solved. They mentioned that some of the links are broken, and I was able to confirm a few. It looks like links that extend across two lines are being tagged as a `<link>` and a `<span>` I tried a few links and the entire second line was missing from the URL in my browser window. Screenshot is from the production URL in the description.

---
![Screen Shot 2020-05-05 at 9 36 49 AM](https://user-images.githubusercontent.com/934879/81078794-4ae4fd80-8eb4-11ea-907a-dbf7ef3d18a4.png)

**Comment by TamGIBill** [comment]
Link Corrected in document.

**Comment by desireeturner** [comment]
@1Copenut This is ready for another review.

**Comment by ERRonat0609** [comment]
Per email from Brian Grubb, Tammy will work on this after first of year (2022)

**Comment by TamGIBill** [comment]
Alt text added and links updated in document.

**Comment by af-manish** [comment]
@TamGIBill I wanted to follow-up to see if you can revisit this.

**Comment by AdamFreemer** [comment]
I initially thought this was a development issue with broken links in code, but the links mentioned are in a PDF document which would fall on being content to update on the VA side.

**Comment by mrblanco1** [comment]
@TamGIBill  - Hi! We conducted analysis and came out to the conclusion we need to determine who can help with this issues. Do you know who could help? Would it be someone on the platform team? Maybe on the EDU side?
Thank you!

**Comment by TamGIBill** [comment]
All 508 issues are handled by the CLAP team and that would be me.  This issue has been resolved.
